docs/reference/ccr/auto-follow.asciidoc

```diff
@@ -7,6 +7,12 @@ each new index in the series is replicated automatically. Whenever the name of
 a new index on the remote cluster matches the auto-follow pattern, a
 corresponding follower index is added to the local cluster.
 
+NOTE: auto-follow patterns creates follower indices only for opened, fully
+assigned indices that match the specified pattern in the remote cluster. The
+ patterns should not match special indices like <<frozen-indices,frozen indices>>,
```

Comment:
> should not

I wonder if this is too strict a wording in light of #67686, at least for searchable snapshots, which we will simply just ignore after #70580? Not sure of frozen and read-only though, I suppose we would fail on them (in which case the wording is fine for those).

Reply:
> Not sure of frozen and read-only though, I suppose we would fail on them (in which case the wording is fine for those).

Yes, auto-follow pattern picks up such indices but the follower indices quickly become red as they cannot be initialized (frozen) or cannot read remote index.

Reply:
I noticed that my first try would be more exact. James helped me to find the right wording and I pushed 4811d5c50adcb968ce1de91260803de4b7083bef. I removed the mention of frozen indices as they might be removed in the future and they are included in indices that blocks writes.